Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -4065,12 +4065,12 @@ static void blendCustomProperty(const CSSPropertyBlendingClient& client, const A
     }
 }
 
-void CSSPropertyAnimation::blendProperty(const CSSPropertyBlendingClient& client, Property property, RenderStyle& destination, const RenderStyle& from, const RenderStyle& to, double progress, CompositeOperation compositeOperation, IterationCompositeOperation iterationCompositeOperation, double currentIteration)
+void CSSPropertyAnimation::blendProperty(const CSSPropertyBlendingClient& client, AnimatableProperty property, RenderStyle& destination, const RenderStyle& from, const RenderStyle& to, double progress, CompositeOperation compositeOperation, IterationCompositeOperation iterationCompositeOperation, double currentIteration)
 {
     WTF::switchOn(property,
         [&] (CSSPropertyID propertyId) {
             blendStandardProperty(client, propertyId, destination, from, to, progress, compositeOperation, iterationCompositeOperation, currentIteration);
-        }, [&] (AtomString customProperty) {
+        }, [&] (const AtomString& customProperty) {
```

Comment:
AtomString is cheap to copy and compare, so I'm not sure this does much. Probably fine though.

Reply:
I was following [previous review feedback](https://github.com/WebKit/WebKit/pull/7780#discussion_r1051307377) from @darinadler.